examples/snap-plugin-collector-rand/rand/rand.go

```diff
@@ -141,6 +174,13 @@ func (RandCollector) GetMetricTypes(cfg plugin.Config) ([]plugin.Metric, error)
 func (RandCollector) GetConfigPolicy() (plugin.ConfigPolicy, error) {
 	policy := plugin.NewConfigPolicy()
 
+	if req {
```

Comment:
No, for the same reason as above. `req` is for testing the case where a config is required in order for the plugin to load. `cfg["depsReq"]` is for testing the case when a plugin has dependencies that are not met.